pkg/operator/edgefs/cluster/target/config/config.go

```diff
@@ -107,6 +115,11 @@ func ToStoreConfig(config map[string]string) StoreConfig {
 			storeConfig.UseAllSSD = convertToBoolIgnoreErr(v)
 		case RtrdPlevelOverrideKey:
 			storeConfig.RtrdPLevelOverride = convertToIntIgnoreErr(v)
+		case SyncKey:
+			value := convertToIntIgnoreErr(v)
+			if value >= 0 && value <= 3 {
+				storeConfig.Sync = convertToIntIgnoreErr(v)
+			}
```

Comment:
I think it would be good to at least log a warning or something if the value is too high as there is currently not CRD validation preventing a bad value here.

Reply:
make sense. done!